OpenXmlPowerTools/DocumentBuilder.cs

```diff
@@ -174,6 +174,12 @@ public Source(WmlDocument source, int start, int count, string insertId)
         }
     }
 
+    public class DocumentBuilderSettings
+    {
+        public List<string> CustomXmlGuidList = null;
```

Comment:
My issue here is that this would be changing the public API.  I would prefer that unless there is a compelling reason, we don't change the public API.

I realize that there are chances that the public API might be improved, but changing the public API probably isn't my first choice.

Reply:
This isn't changing public API as you are creating the class here...

Reply:
Ah, I see what happened here - I made a local change for another user with regards to copying custom XML parts, so this only impacts one other project.  I made it a HashSet instead of a list, which is a good idea.